petab/v1/visualize/plotting.py

```diff
@@ -63,7 +63,10 @@ def __init__(
         self.data_to_plot = data_to_plot
         self.data_to_plot.sort_index(inplace=True)
 
-        self.conditions = conditions_
+        try:
+            self.conditions = conditions_.copy()
+        except AttributeError:
+            self.conditions = conditions_
```

Comment:
This might be more readable
```suggestion
        self.conditions = conditions_
        if self.conditions is not None:
            self.conditions = self.conditions.copy()
```

or just `self.conditions = copy.deepcopy(conditions_)`